.jestrc.json

```diff
@@ -19,13 +19,14 @@
   "moduleFileExtensions": [ "ts", "tsx", "js", "jsx", "json" ],
   "testPathIgnorePatterns": [ "<rootDir>/(build|docs|node_modules|images)/" ],
   "testEnvironment": "jsdom",
-  "testRegex": "\\.test.tsx?$",
+  "testRegex": "\\.test.(ts|tsx|js)$",
```

Comment:
why js when there's only ts files?

Reply:
looks like cucmber is still js?  are we going to migrate that?

Reply:
For JS: I don't know. I did not want to change the current regex since it was out of scope of my work.
For Cucumber: It's out of scope of this work (contact test) and can be opened as a tech-debt item for another time, IMO.

Reply:
I'll try to remove the TS and make sure all the tests still run...

Reply:
JS files are still needed - I suggest we mark it as tech debt for another time as it's OOS for this PR

Reply:
sorry.. it looks like you changed it to include JS files, but you only added TS files, so I don't understand why we need JS files here in this regex.

Reply:
I'm also concerned about how we plan to manage tech debt...